### DISCUSSION TITLE
ease of writing commit messages

### DISCUSSION THREAD

**Discussion by vankhoawin** [twilio-labs/paste#788] [Ideas] [answered]
i was getting rejected for my commit messages not being in the conventional-commit format; frankly i've never had to write it manually before. is there any interest in using a CLI like [commitizen](https://github.com/commitizen/cz-cli) that would guide users through the process?

**Comment by corinne-nagel** [comment, marked as answer]
Hey @vnguyen94 , we like this idea but don't currently have the capacity to work on it. Would you have any interest in contributing it?

**Reply by vankhoawin** [reply]
could look into it sometime in the next few weeks
